ui/types/vault/services/store.d.ts

```diff
@@ -3,16 +3,11 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import Store, { RecordArray } from '@ember-data/store';
+import Store from '@ember-data/store';
```

Comment:
At some point we should be able to remove this declaration file and use https://www.npmjs.com/package/@types/ember-data instead, but when I attempted in this PR it didn't work out out of the box so I abandoned for now. (Especially since EmberData 5.4.0-alpha.35 is published with alpha-level support for typescript, so we may be able to avoid installing another dependency all together)